builds/checkin/e2e-checkin.yaml

```diff
@@ -42,8 +42,8 @@ stages:
       and
       ( 
         succeededOrFailed(),
-        in(dependencies.PublishManifests.result, 'Succeeded','Skipped'),
-        in(dependencies.BuildPackages.result, 'Succeeded','Skipped')
+        in(dependencies.PublishManifests.result, 'Succeeded','Skipped', 'SucceededWithIssues'),
+        in(dependencies.BuildPackages.result, 'Succeeded','Skipped', 'SucceededWithIssues')
```

Comment:
If PublishManifests or BuildPackages returns 'Skipped', won't the `succeededOrFailed()` expression cause the RunE2ETest stage to be skipped? I think `succeededOrFailed()` should be removed, shouldn't it?

Reply:
it would still run since ```succeededOrFailed()``` works for skipped jobs ( It returns false for cancelled jobs). But I agree, it can be removed.